libtransmission/rpc-server.cc

```diff
@@ -51,12 +58,19 @@ using namespace std::literals;
 
 static char constexpr MyName[] = "RPC Server";
 
+tr_rpc_address const tr_rpc_inaddr_any = { TR_RPC_AF_INET, { INADDR_ANY } };
```

Comment:
- this should be static

- _(nit)_ I think this could be constexpr instead of const?